src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -113,6 +127,14 @@ public void setMotorSpeedShoot(double speed) {
     rightMotor.set(ControlMode.PercentOutput, speed);
   }
 
+  public void setInstantRumble() {
+    conDriver.setRumble(RumbleType.kBothRumble, prefControllers.rumbleOutput.getValue());
+    conOperator.setRumble(RumbleType.kBothRumble, prefControllers.rumbleOutput.getValue());
+    Timer.delay(prefControllers.rumbleDelay.getValue());
```

Comment:
I'm pretty sure this will cause all the robot code to wait for this delay. I could be wrong but you should still probably test this. You could use a command wait or the rio FPGA timer instead.